src/app/js/admin/fields/wizard/StepDisplay.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import compose from 'recompose/compose';
+import translate from 'redux-polyglot/translate';
+import { Field } from 'redux-form';
+
+import Step from './Step';
+import FormCheckboxField from '../../../lib/FormCheckboxField';
+import PositionField from '../PositionField';
+import Format from '../../FormatEdition';
+import { polyglot as polyglotPropTypes, field as fieldPropTypes } from '../../../propTypes';
+
+export const StepDisplayComponent = ({
+    field,
+    p: polyglot,
+    ...props
+}) => (
+    <Step label="field_wizard_step_display" {...props}>
```

Comment:
How do we tell which props come from here and which props come from reduxForm. And how to be sure which props is actually needed?
List all props that need to be passed, ignore the rest.